GameData/RealismOverhaul/Engine_Configs/XLR11_Config.cfg

```diff
@@ -37,12 +53,12 @@
 //	XLR11-RM-13-8K
 //	X-15
 //
-//	Dry Mass: 157 Kg
+//	Dry Mass: 212 Kg
 //	Thrust (SL): ??? kN
 //	Thrust (Vac): 38.45 kN
 //	ISP: 208 SL / 225 Vac
```

Comment:
Is there any documentation on the specific impulse of this or the 10K? It seems odd that increasing thrust (presumably increasing chamber pressure slightly?) did not result in any increase in specific impulse at sea level.